src/theory/arrays/array_info.h

```diff
@@ -155,7 +155,7 @@ class ArrayInfo {
   currentStatisticsRegistry()->registerStat(&d_maxList);
   currentStatisticsRegistry()->registerStat(&d_tableSize);
   }*/
-  ArrayInfo(context::Context* c, Backtracker<TNode>* b);
+  ArrayInfo(context::Context* c, Backtracker<TNode>* b, std::string name = "");
```

Comment:
What does an empty name mean? Is this a reasonable default? This constructor probably needs documentation.